src/Auth/IAuthorizationResolver.cs

```diff
@@ -56,6 +56,15 @@ public interface IAuthorizationResolver
     /// <returns></returns>
     public IEnumerable<string> GetAllowedExposedColumns(string entityName, string roleName, EntityActionOperation operation);
 
+    /// <summary>
+    /// Returns true/false depending on whether a database policy is defined for an operation within an entity's role entry.
+    /// </summary>
+    /// <param name="entityName">Entity from request</param>
+    /// <param name="roleName">Role defined in client role header.</param>
+    /// <param name="operation">Operation type: Create, Read, Update, Delete</param>
+    /// <returns>Returns true if a database policy is defined for the operation of a given role. If not, returns false.</returns>
+    public bool IsDBPolicyDefinedForRoleAndAction(string entityName, string roleName, EntityActionOperation operation);
```

Comment:
See comment in `SqlMutationEngine`: you don't need to make this change if you perform the null check on the return of `authZResolver.GetDBPolicyForRequest` from within your biz logic in sqlmutationengine.